src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -945,6 +946,8 @@ class WhisperDecoder(WhisperPreTrainedModel):
         config: WhisperConfig
     """
 
+    main_input_name = "input_ids"
```

Comment:
Was this an oversight in the implementation?

Reply:
`WhisperDecoder` was never tested as stand-alone before. It's also doesn't really make sense to use it alone because on always needs the encoded audio features